src/inrange.jl

```diff
@@ -50,3 +52,33 @@ function inrange(tree::NNTree{V}, point::AbstractMatrix{T}, radius::Number, sort
     end
     inrange(tree, new_data, radius, sortres)
 end
+
+"""
+    inrangecount(tree::NNTree, points, radius) -> count
+
+Count all the points in the tree which is closer than `radius` to `points`.
```

Comment:
```suggestion
Count all the points in the tree which are closer than `radius` to `points`.
```

Reply:
Thanks! To be fair I just copied the `inrange` docstring, so that should get the correction too :-)